nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/TransactionalFeature.java

```diff
@@ -54,7 +56,18 @@
     @FunctionalInterface
     public interface Waiter {
 
-        boolean await(long deadline) throws InterruptedException;
+        /**
+         * @deprecated since 10.3, use {@link #await(Duration)} instead.
+         */
```

Comment:
waiting for a deadline is a different usage than waiting for a timeout but ok for depreciation if the first usage is not is necessary

Reply:
My only motivation was to use `Duration` as it seem to become a standard. Can be rollback if wanted.